src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -34,6 +34,7 @@
         "content-launch-cluster.xml",
         "descriptor-cluster.xml",
         "diagnostic-logs-cluster.xml",
+        "dishwasher-mode-select-cluster.xml",
```

Comment:
This seems to do more than updating modeselect. Could we have a separate PR for this?

Reply:
LaundryWasher is an alias of mode select as are all the others added. This means that they are all similar and related. If you look at the `ModeSelectFeature`, `SemanticTagStruct` and `ModeOptionStruct` defined in the `mode-select-cluster.xml` you will see that the cluster codes for all of these clusters are set here. I think it would be more cumbersome and confusing to review these additions independantly.